azure-pipelines.yml

```diff
@@ -169,45 +169,9 @@ jobs:
       includePatterns:
         - "/sw/***"
 
-# Software targeting the English Breakfast top level is produced by patching
-# the source tree before building. This builds a selected subset of software
-# only.
 # TODO: This is a rather ugly hack, which will go away once we properly support
 # building more than one top-level design with different parametrizations.
 # Work towards this goal is tracked in issue #4669.
```

Comment:
This TODO message is specifically about how we hack the software. I don't *think* it applies to the `chip_englishbreakfast_verilator` target?

Reply:
oops. copy/paste error. thanks!